common/src/main/scala/com/gu/recipeasy/db/DB.scala

```diff
@@ -319,6 +319,32 @@ class DB(contextWrapper: ContextWrapper) {
     )
   }
 
+  def userEventsDates(): List[String] = {
+    contextWrapper.dbContext.run(
+      quote(
+        query[UserEventDB].schema(_.entity("user_events")).map(event => event.event_datetime)
+      )
+    ).map(datetime => {
+        val format = new java.text.SimpleDateFormat("yyyy-MM-dd")
+        format.format(datetime)
+      }).distinct
+  }
+
+  def eventsForDateAndOperationType(date: String, opType: OperationType): List[UserEventDB] = {
+    val format = new java.text.SimpleDateFormat("yyyy-MM-dd")
```

Comment:
You can import `java.text.SimpleDateFormat` seeing as it's being used twice rather than typing the full package name out each time.